src/speech_to_text_finetune/hf_utils.py

```diff
@@ -4,7 +4,18 @@
 import importlib.util
 from pathlib import Path
 from typing import Dict
-from huggingface_hub import ModelCard, HfApi, ModelCardData, hf_hub_download, EvalResult
+from huggingface_hub import (
+    ModelCard,
+    HfApi,
+    ModelCardData,
+    EvalResult,
+    hf_hub_download,
+    repo_exists,
+)
+
+
+def hf_model_exists(model_repo_id: str) -> bool:
+    return repo_exists(model_repo_id)
```

Comment:
This feels unnecessary

Reply:
Yeah, I was thinking I would make the huggingface package optional to install and only import that function if the user goes through the HF path, but I dont think it makes much sense to add code complexity for this reason. Removed.